requirements.apt

```diff
@@ -0,0 +1,9 @@
+libignition-cmake2-dev
```

Comment:
try and specify some versions in this file

Reply:
done